trieTools/trieChecker/main.go

```diff
@@ -28,7 +28,7 @@ func main() {
 	app := cli.NewApp()
 	app.Name = "Trie checker CLI app"
 	app.Usage = "This is the entry point for the tool that checks the trie DB"
-	app.Flags = getFlags()
+	app.Flags = trieToolsCommon.GetFlags()
```

Comment:
update `app.Authors` for other tools as well

Reply:
Done.